### ISSUE TITLE
How to install Vulken in debian?

### ISSUE THREAD

**Issue by akashpawar1** [lutris/docs#134]
I am trying to install in Debian using command for ubuntu:
`sudo add-apt-repository ppa:kisak/kisak-mesa && sudo dpkg --add-architecture i386 && sudo apt update && sudo apt upgrade && sudo apt install libgl1-mesa-dri:i386 mesa-vulkan-drivers mesa-vulkan-drivers:i386`

but it shows:
`sudo: add-apt-repository: command not found`

how can I install driver in Debian

**Comment by financelurker** [comment]
yeah, especially since https://wiki.debian.org/DontBreakDebian highly suggests NOT to install ubuntu PPAs, since they can lead to an unsupported, corrupt FrankenDebian... I'd love to see a debian-native way (which ubuntu and other debian-derivates) could use ootb.

**Comment by ACR-Jeff** [comment]
I have complained about this very subject for years on this, Yet get criticized or ignored, I had to find out that Debian preinstall's Vulkan drivers automatically upon install, Yet now they force the warnings and graying out settings in Runner options even though you may already have Vulkan drivers, DXVK and VKD3D installed, The settings being grayed out and nonoperational tend to break things in lutris further. Yet they focus only on Ubuntu based distro's and forget that Ubuntu is a Debian Based Fork.

**Comment by strycore** [comment]
I have added a note on the "Installing drivers" page that the Lutris team does not endorse the instructions on this page.

No distribution needs a PPA to get Vulkan. The only gotcha is that you have to install 64bit *and* 32bit versions of some libraries.

**Comment by dpanter** [comment]
Information from [Debian wiki](https://wiki.debian.org/Steam) about Steam.
Steps 2 and 5 are directly relevant for GPU drivers. Use sudo or similarly elevated permissions to install.


___2. For amd64 (64-bit) systems, enable [Multi-Arch](https://wiki.debian.org/Multiarch/HOWTO):___
```
# dpkg --add-architecture i386
```
___5. For amd64 (64-bit) systems, additional libraries need to be installed for Vulkan and 32-bit titles:___
```
# apt install mesa-vulkan-drivers libglx-mesa0:i386 mesa-vulkan-drivers:i386 libgl1-mesa-dri:i386
```
